module/renderer/src/commonMain/kotlin/org/cru/godtools/shared/renderer/content/RenderTabs.kt

```diff
@@ -0,0 +1,72 @@
+@file:OptIn(ExperimentalMaterial3Api::class)
+
+package org.cru.godtools.shared.renderer.content
+
+import androidx.compose.foundation.background
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.SecondaryTabRow
+import androidx.compose.material3.Tab
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import org.cru.godtools.shared.renderer.state.State
+import org.cru.godtools.shared.tool.parser.model.Tabs
+
+private val TAB_SELECTED_COLOR = Color.DarkGray
+private val TAB_UNSELECTED_COLOR = Color.LightGray
+private val TAB_SELECTED_CONTENT_COLOR = Color.White
+private val TAB_UNSELECTED_CONTENT_COLOR = Color.DarkGray
+
+@Composable
+internal fun RenderTabs(tabs: Tabs, state: State, modifier: Modifier = Modifier) {
+    val selectedIndex = remember { mutableStateOf(0) }
+
+    SecondaryTabRow(
+        selectedTabIndex = selectedIndex.value,
+        modifier = modifier,
+        containerColor = Color.White,
+        contentColor = Color.White,
```

Comment:
the colors for tabs have historically have been pulled from `tabs.primaryColor` and `tabs.primaryTextColor`

Reply:
where it had been defined in Android previously:

https://github.com/CruGlobal/godtools-android/blob/develop/ui/base-tool/src/main/res/layout/tool_content_tabs.xml#L24-L27
https://github.com/CruGlobal/godtools-android/blob/develop/ui/base-tool/src/main/kotlin/org/cru/godtools/base/tool/ui/controller/TabsController.kt#L81